packages/react/src/components/Tabs/Tabs.tsx

```diff
@@ -2,9 +2,9 @@ import { Cauldron } from '../../types';
 import React, { useState, useEffect, useRef } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import { useDidUpdate } from '../../index';
 import Tab from './Tab';
 import { useId } from 'react-id-generator';
+import useDidUpdate from '../../utils/use-did-update';
```

Comment:
We were getting a warning about a cyclic dependency. I made a quick tweak to address it.